src/main/java/io/ebean/config/dbplatform/oracle/OracleDbEncrypt.java

```diff
@@ -13,12 +13,11 @@
  *
  * <pre class="code">
  *
- *  // Remember your DB user needs execute privilege on DBMS_CRYPTO
```

Comment:
This is the result of having Intellij accept the editorconfig file as default for the project and then autoformatting the entire project.